ThreadGroup/Controllers/TGNavigationViewController.m

```diff
@@ -235,4 +236,11 @@ - (UIAlertController *)moreMenu {
     return _moreMenu;
 }
 
+#pragma mark - UINavigationControllerDelegate
+
+- (id<UIViewControllerAnimatedTransitioning>)navigationController:(UINavigationController *)navigationController animationControllerForOperation:(UINavigationControllerOperation)operation fromViewController:(UIViewController *)fromVC toViewController:(UIViewController *)toVC {
+    TGNavigationAnimator *animator = [TGNavigationAnimator new];
+    return animator;
+}
```

Comment:
Also, use the pattern you have in the UINavigationController delegate method in `TGRootVC'